### PR TITLE
Add go live template (Migration)

### PR DESCRIPTION
**Pull request by imdadahad**
This adds a template in a migration that will be sent automatically when a service goes live. 

_Note: This will also include some personal information that'll be added separately._

### CODE REVIEW

File: migrations/versions/0082_add_golive_template.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+^Only use this email address for out of hours emergencies. Don’t share this address with people outside of your team.\n\n
+
+We’ll get back to you within 30 minutes and give you hourly updates until the problem’s fixed.\n\n
+
+For non-emergency problems or questions, use our support page and we’ll reply in office hours:
+https://www.notifications.service.gov.uk/support \n\n
+#Escalation for emergency problems\n\n
+
+If we haven’t acknowledged an emergency problem you’ve reported within 30 minutes and you need to know what’s happening, you can escalate to:\n\n
+
```
Comment (quis) [inline migrations/versions/0082_add_golive_template.py]: Something missing here?

**Comment by quis** [comment]
I don’t think you need all the `\n`s if you’re `"""`triple quoting`"""` stuff.

File: migrations/versions/0082_add_golive_template.py (hunk 1 above)
Comment (imdadahad) [inline migrations/versions/0082_add_golive_template.py]: Ah yes, sorry forgot to update the description.

I'm planning on inserting personal content (phone number etc.) manually. I didn't want those details to be in git history.

Comment (quis) [inline migrations/versions/0082_add_golive_template.py]: Good point. Didn’t think about that.

**Comment by imdadahad** [comment]
Thanks, just removed them.

**Pull request merged**